proposals/lip-0057.md

```diff
@@ -930,7 +930,7 @@ pomParams = {
 
 ##### Verification
 
-Both properties of the parameters must follow the [block header schema `blockHeaderSchema`][lip-0055#block-header-json-schema] defined in LIP 0055. Validity of this transaction was previously specified in [LIP 0024][lip-0024#validity-of-a-pom-transaction]. For completeness, we include the pseudocode here.
+Both properties of the parameters must follow the [block header schema `blockHeaderSchema`][lip-0055#block-header-json-schema] defined in LIP 0055. Validity of this transaction was previously specified in [LIP 0024][lip-0024#validity-of-a-misbehaviorReport-transaction]. For completeness, we include the pseudocode here.
```

Comment:
A link to LIP 24 is broken here. I guess, no changes are needed here, it should stay `validity-of-a-pom-transaction`.

Reply:
Good catch, fixed. I kept the old links at the end of the file, but some of them in the text were overlooked.